integrationTests/vm/arwen/arwenVM/gasSchedule_test.go

```diff
@@ -0,0 +1,236 @@
+}
+
+func Benchmark_TestBigIntMul25(b *testing.B) {
+	runWASMVMBenchmark(b, "../testdata/c-api-tests/bigInt/output/cApiTest.wasm", 0, "bigIntMul25Test", nil, b.N, nil)
+}
+
+func Benchmark_TestBigIntMul32(b *testing.B) {
+	runWASMVMBenchmark(b, "../testdata/c-api-tests/bigInt/output/cApiTest.wasm", 0, "bigIntMul32Test", nil, b.N, nil)
+}
+
+func Benchmark_TestBigIntTDiv(b *testing.B) {
+	runWASMVMBenchmark(b, "../testdata/c-api-tests/bigInt/output/cApiTest.wasm", 0, "bigIntTDivTest", nil, b.N, nil)
+}
+
+func Benchmark_TestBigIntTMod(b *testing.B) {
+	runWASMVMBenchmark(b, "../testdata/c-api-tests/bigInt/output/cApiTest.wasm", 0, "bigIntTModTest", nil, b.N, nil)
+}
+
+func Benchmark_TestBigIntEDiv(b *testing.B) {
+	runWASMVMBenchmark(b, "../testdata/c-api-tests/bigInt/output/cApiTest.wasm", 0, "bigIntEDivTest", nil, b.N, nil)
+}
+
+func Benchmark_TestBigIntEMod(b *testing.B) {
+	runWASMVMBenchmark(b, "../testdata/c-api-tests/bigInt/output/cApiTest.wasm", 0, "bigIntEModTest", nil, b.N, nil)
+}
+
+func Benchmark_TestBigIntShr(b *testing.B) {
+	runWASMVMBenchmark(b, "../testdata/c-api-tests/bigInt/output/cApiTest.wasm", 0, "bigIntShrTest", nil, b.N, nil)
+}
+
+func Benchmark_TestBigIntSetup(b *testing.B) {
+	runWASMVMBenchmark(b, "../testdata/c-api-tests/bigInt/output/cApiTest.wasm", 0, "bigIntInitSetup", nil, b.N, nil)
+}
+
+func Benchmark_TestCryptoSHA256(b *testing.B) {
+	runWASMVMBenchmark(b, "../testdata/c-api-tests/crypto/output/cryptoTest.wasm", 0, "sha256Test", nil, b.N, nil)
+}
+
+func Benchmark_TestCryptoKeccak256(b *testing.B) {
+	runWASMVMBenchmark(b, "../testdata/c-api-tests/crypto/output/cryptoTest.wasm", 0, "keccak256Test", nil, b.N, nil)
+}
+
+func Benchmark_TestCryptoRipMed160(b *testing.B) {
+	runWASMVMBenchmark(b, "../testdata/c-api-tests/crypto/output/cryptoTest.wasm", 0, "ripemd160Test", nil, b.N, nil)
+}
+
+func Benchmark_TestCryptoBLS(b *testing.B) {
+	runWASMVMBenchmark(b, "../testdata/c-api-tests/crypto/output/cryptoTest.wasm", 0, "verifyBLSTest", nil, b.N, nil)
+}
+
+func Benchmark_TestCryptoVerifyED25519(b *testing.B) {
+	runWASMVMBenchmark(b, "../testdata/c-api-tests/crypto/output/cryptoTest.wasm", 0, "verifyEd25519Test", nil, b.N, nil)
+}
+
+func Benchmark_TestCryptoSecp256k1UnCompressed(b *testing.B) {
+	runWASMVMBenchmark(b, "../testdata/c-api-tests/crypto/output/cryptoTest.wasm", 0, "verifySecp256k1UncompressedKeyTest", nil, b.N, nil)
+}
+
+func Benchmark_TestCryptoSecp256k1Compressed(b *testing.B) {
+	runWASMVMBenchmark(b, "../testdata/c-api-tests/crypto/output/cryptoTest.wasm", 0, "verifySecp256k1CompressedKeyTest", nil, b.N, nil)
+}
+
+func Benchmark_TestCryptoDoNothing(b *testing.B) {
+	runWASMVMBenchmark(b, "../testdata/c-api-tests/crypto/output/cryptoTest.wasm", 0, "doNothing", nil, b.N, nil)
+}
+
+func Benchmark_TestStorageRust(b *testing.B) {
+	gasSchedule, _ := core.LoadGasScheduleConfig("../gasSchedule.toml")
```

Comment:
Should this benchmark be run with `gasScheduleV2.toml`? Or using the old gas schedule was intentional, for comparison?

Reply:
StorageTest had good values with V1 as well so did not change it as the difference was small.